plugin/plugin.go

```diff
@@ -151,13 +157,13 @@ func Install(w *app.Window, evt event.Event) {
 	}
 
 	switch evt.(type) {
-	case system.FrameEvent:
-		ref := *(**system.FrameEvent)(unsafe.Add(unsafe.Pointer(&evt), unsafe.Sizeof(uintptr(0))))
+	case app.FrameEvent:
+		ref := *(**app.FrameEvent)(unsafe.Add(unsafe.Pointer(&evt), unsafe.Sizeof(uintptr(0))))
 		h.invalidated.Store(false)
 
-		q := ref.Queue
-		h.queue = q
-		ref.Queue = h
+		// q := ref.Queue
+		// h.queue = q
+		// ref.Queue = h
```

Comment:
The `h` is the `plugin`, that unsafe hack is used to make possible to read/write events to gio-plugins, using the same API of Gio. Comment it makes impossible to read/write command.